### PR TITLE
Environment Constraint updated

### PR DESCRIPTION
**Pull request by yyht6300a**

### CODE REVIEW

File: Userrequirement/EnvironmentConstrait.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+Time limitation:
+
+
+Because our group reoriented the first milestones to build a website outside the game, 
+we don’t have too much time left to finish coding. In the next three weeks, 
+we need to create a website that contains our lessons, a database for users to store their 
+data and test the connection between web and Minecraft.
```
Comment (Ansper11) [inline Userrequirement/EnvironmentConstrait.txt:12]: Yes totally agree

**Comment by Ansper11** [comment]
Thanks for bringing a detailed User group, nice job.

File: Userrequirement/UserGroup.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
+User Groups
+
+
+
+Teachers (primary):
+
+
+Our project is trying to provide a set of lessons for instructors who are 
+trying to teach their students AI and ML. Based on students of different ages, 
+there are appropriate curriculums for them. Teaching students coding for a game
```
Comment (Nick-McGee) [inline Userrequirement/UserGroup.txt:10]: This is very true!

File: Userrequirement/EnvironmentConstrait.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
+Environmental constraints
+
+
+
+
+Time limitation:
+
+
+Because our group reoriented the first milestones to build a website outside the game, 
+we don’t have too much time left to finish coding. In the next three weeks,
```
Comment (Nick-McGee) [inline Userrequirement/EnvironmentConstrait.txt:10]: It may be two weeks

**Comment by yyht6300a** [comment]
Anything else you guys want to mention in our report?

**Pull request merged**